string-manipulation-2/src/first-chars.js

```diff
@@ -0,0 +1,21 @@
+/* exported firstChars */
+function firstChars(length, string) {
+  var container = '';
+  for (var i = 0; i < length; i++) {
+    if (string[i] === undefined) {
+      break;
+    }
+    container += string[i];
+  }
+  if (string === '') {
+    return container;
+  }
```

Comment:
I recommend doing this right after `container` is declared. It's good to get the special cases like this out of the way near the top of the function